cmd/osquery-perf/agent.go

```diff
@@ -391,6 +392,7 @@ type softwareInstaller struct {
 
 func newAgent(
 	agentIndex int,
+	hostCount int,
```

Comment:
This is set to `500` in the loadtest environment (and we spin up multiple of these to get to the desired host count).

Reply:
Double checking you are not expecting this to be the total number of hosts for the load test.

Reply:
Double checking, are you expecting `hostCount` to be e.g. `100000` on the test we will run on the load test environment? If so, we'll need a new flag in osquery-perf something like `--total_host_count 100000`.

Reply:
If I specify `--host_count 100000`, I expect there to be `100000` hosts to be generated.

Reply:
I see, the problem with the load test environment is we are creating 100 containers each with 500 hosts. This generates 50,000 hosts. The problem with creating a new flag called `total_host_count`, is that each container doesn't know which out of 100 it is.
The logic I use to generate the right slice of software is `startIdx := (a.agentIndex - 1) * perHostCount` where agentIndex is the host number. if we have 100 containers, this agent index will be 0-500 on each 100 containers. We almost need to know which container out of 100 we are in, that way we can multiple the agentIndex by the container number.